docs/package-lock.json

```diff
@@ -1,8 +1,3128 @@
 {
   "name": "label-studio-documentation",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
please never commit this file! 
don't use git commit -a, use git commit docs/source/guide/file-to-commit.md instead of  -a